src/Core/Entities/Role.cs

```diff
@@ -1,9 +1,11 @@
-﻿namespace Bit.Core.Entities;
+﻿#nullable enable
+
+namespace Bit.Core.Entities;
 
 /// <summary>
 /// This class is not used. It is implemented to make the Identity provider happy.
 /// </summary>
 public class Role
 {
-    public string Name { get; set; }
+    public string? Name { get; set; }
```

Comment:
I don't want to edit this weird unused class...

Reply:
I removed the annotation but had to add a `#nullable disable` to the file so that it doesn't get turned on once we turn on nullable for the whole project.